src/sidebar/components/filter-status.js

```diff
@@ -0,0 +1,156 @@
+import { createElement } from 'preact';
+
+import { countVisible } from '../util/thread';
+
+import Button from './button';
+
+import useRootThread from './hooks/use-root-thread';
+import useStore from '../store/use-store';
+
+/**
+ * Render information about the currently-applied filters and allow the
+ * user to clear filters or toggle user-focus mode on and off
+ *
+ * When any filtering is applied, we are in one of 3 mutually-exclusive filtering
+ * "modes". In any mode, we display descriptive text about what the user
+ * is looking at, and present one button to take action on the current filtering
+ * state:
+ * - selection: One or more annotations have been selected in the document by
+ *   the user, or we're looking at a direct-linked annotation. Button clears
+ *   the selection. This mode supersedes all others (i.e. other filters ignored).
```

Comment:
The comment says "this mode supersedes all others" but in a `switch` block below there is code that changes the text in `selection` mode depending on whether focus mode is active.

Reply:
Similar to my previous comment: there's a confusion here (not surprising as this stuff is complex) between this component's notion of what "focus mode" means and "user focus mode" overall.